File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://training-support.net/webelements/dynamic-controls")
    ## # Print the title of the page
    print("Title of the page: ", driver.title)
    
    ### Find the checkbox
    checkbox = driver.find_element(By.ID, "checkbox")
    ## Find the checkbox toggle button
    toggle_checkbox = driver.find_element(By.XPATH, "//button[text()='Toggle Checkbox']")
    
    ### Verify if the checkbox is displayed or not
    print("Checkbox is displayed: ", checkbox.is_displayed())

    ##Click the checkbox_toggle button to hide checkbox
    toggle_checkbox.click()
    ##Verify again if the checkbox is displayed or not
    print("Checkbox is visible: ", checkbox.is_displayed())
    
    ##Close the browser
    driver.quit()

File: Selenium/Activities/Activity5.java
package Activity;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {
	
	public static void main(String[] args) {
		
		WebDriver driver = new FirefoxDriver();
		
		//the title of the page and print it to the console
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("Title of the page: " + driver.getTitle());
		
		//Find the checkbox input element
		WebElement checkbox = driver.findElement(By.id("checkbox"));
		//
		driver.findElement(By.xpath("//button[text()='Toggle Checkbox']")).click();
		
		System.out.println("Checkbox is displayed: " + checkbox.isDisplayed());
		
		driver.findElement(By.xpath("//button[text()='Toggle Checkbox']")).click();
				
		System.out.println("Checkbox is displayed: " + checkbox.isDisplayed());
		
		//Close the browser
		driver.quit();
	}

}
